apps/builder/app/shared/copy-paste/utils/init-copy-paste.ts

```diff
@@ -74,7 +55,7 @@ export const startCopyPaste = <Type>(props: Props<Type>) => {
 
     // must prevent default, otherwise setData() will not work
     event.preventDefault();
-    event.clipboardData.setData("application/json", serialize(data));
+    event.clipboardData.setData(format, stringify(data));
```

Comment:
stringily and paste may not be necessary as configuration. onCut and onCopy may return already serialized value and onPaste parse it in place.

Reply:
true